_data/speakers.yml

```diff
@@ -68,9 +68,18 @@
   name: "Dorai"
   surname: "Thodla"
   company:
-    - {name: "TBA", title: "TBA"}
-  bio: "TBA"
-  thumbnailUrl:
+    - {name: "iMorph Inc.", title: "Founder"}
+    - {name: "Future Focus/Focus America", title: "CTO"}
+  bio: " Dorai Thodla has over 40 years experience in Software Industry and he has founded 4 product startups - 2 in India and 2 in USA. Currently he is working in below roles:<br>
```

Comment:
yes, it works perfectly. Please refer to screenshot -  http://i.imgur.com/1XPEcQH.png